### PR TITLE
Stripe Recipient Create Options Update to Include Debit Card. Stripe Transfer Options Update to Include Recipient Card. Stripe Card Service Update to include Recipient URL.

### PR DESCRIPTION
**Pull request by Giovani-Brady**

### CODE REVIEW

**Comment by jaymedavis** [comment]
can you summarize what all is going on here?

a couple of things off hand:

1) I'm not a big fan of the properties, I'd rather just not repeat typing the same thing if that's possible.
2) The interface option just means you would have to implement that in a lot of different places, probably why you did the number 1. There are lots of classes that would need this interface.

**Comment by Giovani-Brady** [comment]
Updated implementation.

Stripe API Recipient Services supports the use of a debit card. (https://stripe.com/docs/api#create_recipient) Updated StripeRecipientCreateOptions and StripeRecipientUpdateOptions to include card properties. (https://stripe.com/docs/api#create_card) Stripe API Card Services support CRUD opertions of Customer credit cards and Recipient Debit Cards. Updates to StripeCardService to support these operations. Stripe API Transfer Services support transfers to recipient bank accounts and debit cards. (https://stripe.com/docs/api#create_transfer) Updated StripeTransferCreateOptions to include card property.

**Comment by jaymedavis** [comment]
I just pushed a big update (2.7.0) that addresses the majority of this. Let me know if anything is missing!

Bank accounts and cards are now supported together. Thanks for your initial pull request, but I had something in flight already to solve the problem. :)

**Comment by Giovani-Brady** [comment]
2.7.0 updates are missing StripeCardService Recipient debit card create, retrieve, and update API endpoints. This commit includes those changes.

**Comment by jaymedavis** [comment]
Can you be more specific?

**Comment by Giovani-Brady** [comment]
ending thread.
